### ISSUE TITLE
[Designer Accessibility] Drag designer peer button should be keyboard accessible

### ISSUE THREAD

**Issue by anna-dingler** [microsoft/AdaptiveCards#7795]
Keyboard accessibility needs to be enabled for the drag button on designer peers.

<img width="404" alt="image" src="https://user-images.githubusercontent.com/98650930/187735433-55c05165-5c49-47bd-b02d-b4fe8701c63f.png">

See https://github.com/microsoft/AdaptiveCards/pull/7159
